frontend/src/app/fakeData/patients.js

```diff
@@ -1,7 +1,8 @@
 // dummy responses for async queries
 export const demoPatients = [
   {
-    patientId: "patientId1",
+    patientId: "$PATIENT-b3xkhkyxb70",
```

Comment:
This is the form the new internal `patientId` takes. I'm assuming we'll need to update this further when we connect with the backend but this gets us up and working for the moment.